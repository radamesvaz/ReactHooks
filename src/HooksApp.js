import React from 'react'

export const HooksApp = () => {
    return (
        <>
            <h1>Hola Mundo</h1>
        </>
    )
}
