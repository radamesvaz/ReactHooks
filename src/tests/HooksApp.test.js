import React from 'react';
import { shallow } from "enzyme"
import { HooksApp } from "../HooksApp"

describe('Haciendo pruebas sobre el componente <HookApp />', () => {
    
    test('Debe de mostrar el componente correctamente ', () => {
        
        const wrapper = shallow(<HooksApp />);

        expect(wrapper).toMatchSnapshot();
    })
    
})
