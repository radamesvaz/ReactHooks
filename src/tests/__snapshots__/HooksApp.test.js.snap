// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Haciendo pruebas sobre el componente <HookApp /> Debe de mostrar el componente correctamente  1`] = `
<Fragment>
  <h1>
    Hola Mundo
  </h1>
</Fragment>
`;
